src/main/java/com/stripe/model/PaymentIntent.java

```diff
@@ -134,7 +134,7 @@
    * fulfill a payment using the provided source.
    */
   @SerializedName("next_source_action")
-  PaymentIntentSourceAction nextSourceAction;
+  PaymentIntentNextSourceAction nextSourceAction;
```

Comment:
This is unrelated to the review (or a change in openapi). Is this because of some client-metadata changes?
Asking out of curiosity, this should go away since we're now doing `next_action` instead anyway in the next API version

Reply:
Changed to nested `NexAction`